docker-compose-dev.yml

```diff
@@ -61,5 +61,18 @@ services:
       - './.localstack:/tmp/localstack'
       - $PWD/localstack:/docker-entrypoint-initaws.d
 
+  # Optional - only when running API locally
+  licences-db:
+    image: postgres
+    networks:
+      - hmpps
+    container_name: licences-db
+    restart: always
+    ports:
+      - "5433:5432"
+    environment:
```

Comment:
The port might be an issue - as the API expects it on 5432. Is this because you also have another postgres locally? Also, 5433 is used by other devs to connect to remote K8s db's via port-forwards. Any change we could change to default 5432 here and stop/start other competing pgSQLs please?

Reply:
Yes this makes sense, it should be the default port. I'll make a push now.